tests/test_utils/mock_operators.py

```diff
@@ -166,3 +166,13 @@ class MockHiveOperator(HiveOperator):
     def __init__(self, *args, **kwargs):
         self.run = mock.MagicMock()
         super().__init__(*args, **kwargs)
+
+
+class DeprecatedOperator(BaseOperator):
+    @apply_defaults
+    def __init__(self, *args, **kwargs):
+        warnings.warn("This operator is deprecated.", DeprecationWarning, stacklevel=4)
```

Comment:
The key is the `stacklevel=4`. I'm wondering how can we make sure that all deprecation warnings in operators are using the same stack level. Should we have a custom function? @potiuk @mik-laj @kaxil

Reply:
As we discussed on Slack. We should add an assertion that checks if the warning has a good file

Reply:
Added